R/theme_gtsummary_roche.R

```diff
@@ -32,19 +33,52 @@ theme_gtsummary_roche <- function(set_theme = TRUE, font_size = NULL) {
       )
     )
 
+
   # use arial font for flextables ----------------------------------------------
   lst_theme$`as_flex_table-lst:addl_cmds` <-
     c(
       lst_theme$`as_flex_table-lst:addl_cmds`,
-      list(valign = rlang::expr(flextable::font(fontname = "Arial", part = "all")))
+      list(
+        user_added1 = rlang::expr(flextable::font(fontname = "Arial", part = "all")),
+        user_added1 = rlang::expr(flextable::fontsize(size = 7, part = "footer"))#,
```

Comment:
@ddsjoberg This PR atm is mainly trials. I am wondering how to proceed as I do not know how general themes are in {gtsummary}. Some of these are very specific to docx output (and do not look too cool to me) so I thought, that it could be added to `gtsummary::as_flex_table` as an additional modification for export or if we should actually change it here for the general fruition. If you prefer a general theme setting, I do not know exactly how to make this access variables like border defaults and so on.

Besides my scrambling, I would just meet up when you have time so we can decide how to proceed here (probably faster ^^)